### PR TITLE
Control 2.2.1

### PR DESCRIPTION
**Pull request by TheDauntless**
Proposal for #906 

I only included the 'per hour' and not the total amount, since that one does not have an obvious 'reset' and it would be confusing to a user if they are at '99 failed attempts' for total lifetime, and then suddenly their account is blocked after a single typo.

5 attempts per hour could trigger a response, and typically that means sending an email to the end user allowing them to still log in, which will prevent account lockout as well.

### CODE REVIEW

**Comment by tghosth** [comment]
@elarlang please review and let me know what you think

File: 5.0/en/0x11-V2-Authentication.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ NIST considers SMS as ["restricted" authenticator types](https://pages.nist.gov/
...
-| **2.2.1** | Verify that anti-automation controls are effective at mitigating breached credential testing, brute force, and account lockout attacks. Such controls include blocking the most common breached passwords, soft lockouts, rate limiting, CAPTCHA, ever increasing delays between attempts, IP address restrictions, or risk-based restrictions such as location, first login on a device, recent attempts to unlock the account, or similar. Verify that no more than 100 failed attempts per hour is possible on a single account. | ✓ | ✓ | ✓ | 307 | 5.2.2 / 5.1.1.2 / 5.1.4.2 / 5.1.5.2 |
+| **2.2.1** | Verify that anti-automation controls are effective at mitigating breached credential testing, brute force, and account lockout attacks. Such controls include blocking the most common breached passwords, soft lockouts, rate limiting, CAPTCHA, ever increasing delays between attempts, IP address restrictions, or risk-based restrictions such as location, first login on a device, recent attempts to unlock the account, or similar. More than 5 failed authentication attempts per hour should trigger some sort of reaction or alert. | ✓ | ✓ | ✓ | 307 | 5.2.2 / 5.1.1.2 / 5.1.4.2 / 5.1.5.2 |
```
Comment (tghosth) [inline 5.0/en/0x11-V2-Authentication.md:60]: ```should trigger some sort of reaction or alert.```

Based on @elarlang's question in #906, should this be:
- per user?
- per application?
- per IP?

**Review by tghosth — CHANGES_REQUESTED** [review]

File: 5.0/en/0x11-V2-Authentication.md (hunk 1 above)
Comment (jmanico) [reply 5.0/en/0x11-V2-Authentication.md:60]: I think 5 failed attempts per user is the goal here.

**Comment by TheDauntless** [comment]
Definitely per user, so I've added it to the PR.

**Comment by jmanico** [comment]
Folks, I think we are ready for a PR

**Review by tghosth — APPROVED** [review]

**Pull request merged**